12/umbraco-cms/reference/content-delivery-api/README.md

```diff
@@ -578,7 +578,7 @@ GET /umbraco/delivery/api/v1/content?fetch=children:dc1f43da-49c6-4d87-b104-a586
 {% endtab %}
 
 {% tab title="filter" %}
-The `filter` query parameter allows you to specify one or more filters that must match in order for a content item to be included in the response. The API provides two built-in filters that you can use right away with the `/umbraco/delivery/api/v1/content` endpoint:
+The filter query parameter allows you to specify one or more filters. These filters must match in order for a content item to be included in the response. The API provides a few built-in filters that you can use right away with the `/umbraco/delivery/api/v2/content` endpoint:
```

Comment:
If I remember correctly, for v12 the api versions is v1. Will need to check this, so will confirm in a bit
```suggestion
The filter query parameter allows you to specify one or more filters. These filters must match in order for a content item to be included in the response. The API provides a few built-in filters that you can use right away with the `/umbraco/delivery/api/v1/content` endpoint:
```

Reply:
yeps, it is v1 for v12 so the content here is correct: 
![image](https://github.com/umbraco/UmbracoDocs/assets/83591955/a096df03-1c7e-4b71-994e-3fd8f0f3a845)

You can also check on a v12 project dev env or live env cloned down by going to {{your project URL}}/umbraco/swagger 😊